rcl_interfaces/msg/FloatingPointRange.msg

```diff
@@ -1,14 +1,13 @@
 # Represents bounds and a step value for a floating point typed parameter.
 
-# Lower bound for valid values, inclusive.
-double minimum_value
+# Start value for valid values range, inclusive.
+double from_value
 
-# Upper bound for valid values, inclusive.
-double maximum_value
+# End value for valid values range, inclusive.
+double to_value
 
-# Size of valid steps between the lower and upper bound.
+# Size of valid steps between the from and to bound.
 # Step should be positive, and a step of zero implies continuous values.
-# Step size should be less than the distance between the minimum and maximum
-# values, and ideally would be an even multiple of that distance, but that
-# is not required.
+# Step size should be less than the distance between the values.
+# The to_value is valid even if the step is not a multiple of it.
```

Comment:
What happens if step is larger than the distance?

What about the examples from my proposed doc block (https://github.com/ros2/rcl_interfaces/pull/74#discussion_r277856900)?